DragaliaAPI.Database/Migrations/20230207152134_develop_7.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace DragaliaAPI.Database.Migrations
+{
+    /// <inheritdoc />
+    public partial class develop7 : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_PlayerDragonGift_Players_OwnerAccountId",
+                table: "PlayerDragonGift");
+
+            migrationBuilder.DropIndex(
+                name: "IX_PlayerDragonGift_OwnerAccountId",
+                table: "PlayerDragonGift");
+
+            migrationBuilder.DropColumn(
+                name: "OwnerAccountId",
+                table: "PlayerDragonGift");
+
+            migrationBuilder.RenameColumn(
+                name: "StoryState",
+                table: "PlayerStoryState",
+                newName: "State");
```

Comment:
is it possible to remove this entirely from the migrations? i.e squash the ones you have together?

also your migrations aren't following the new naming scheme based on the version -- i will soon merge in 20230205020012_1-1-0_alpha_6 so could you go with 1-1-0_alpha_7 please